src/bluesearch/database/topic.py

```diff
@@ -0,0 +1,228 @@
+# Blue Brain Search is a text mining toolbox focused on scientific use cases.
+#
+# Copyright (C) 2020  Blue Brain Project, EPFL.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+"""Utils for journal/articles topics."""
+import pathlib
+from typing import Dict, Iterable, List, Union
+from xml.etree.ElementTree import Element  # nosec
+
+import requests
+from defusedxml import ElementTree
+
+
+# Journal Topic
+def get_mesh_from_nlm_ta(nlm_ta: str) -> List[Dict[str, Union[str, List[str]]]]:
+    """Retrieve Medical Subject Heading from Journal's NLM Title Abbreviation.
+
+    Parameters
+    ----------
+    nlm_ta : str
```

Comment:
Thanks, I am always struggling with the type annotations! It is nice we discussed that in Slack. I think it can be a good idea to put it in `CONTRIBUTING.md`.

Change done in 0fca7c3.